scripts/import/dbSNP_v2/load_dbsnp.pl

```diff
@@ -1632,6 +1668,17 @@ sub configure {
       die("ERROR: FASTA file does not exist ($config->{'fasta_file'})\n");
     }
   }
+
+  if ($config->{'assign_ancestral_allele'}) {
+    if (! defined $config->{'ancestral_fasta_file'}) {
+      pod2usage({ -message => "Mandatory argument (ancestral_fasta_file) is missing", 
```

Comment:
But the file is mandatory if --assign_ancestral_allele is set.

Reply:
If you run load_dbsnp.pl without  options --ancestral_fasta_file or --no_assign_ancestral_allele get the message
Mandatory argument (ancestral_fasta_file) is missing
ancestral_fasta_file is not mandatory, the option --no_assign_ancestral_allele could be specified instead of ancestral_fasta_file